src/audiomixerboard.h

```diff
@@ -287,6 +287,9 @@ class CAudioMixerBoard : public QGroupBox, public CAudioMixerBoardSlots<MAX_NUM_
     template<unsigned int slotId>
     inline void connectFaderSignalsToMixerBoardSlots();
 
+    // If iChannelIdx is I_MY_CHANNEL it means iMyChannelID should be used instead
```

Comment:
Clarify _why_ I_MY_Channel must be used instead.

Reply:
It doesn't _have to be_ used.  _By definition_, if you use `I_MY_CHANNEL` for `iChannel`, it means you want `iMyChannel` used instead.  That's just what it _means_.  I'm not sure what else I could say.

Reply:
I meant more of an explanation why this function exists. So say that this constant is used for the new flag introduced here.

Reply:
I've added a long explanation.